assets/samplesheet.csv

```diff
@@ -0,0 +1,3 @@
+sample,fastq_1,fastq_2
```

Comment:
This file is used as an example of how to provide samples to the pipeline using a `samplesheet.csv`

Reply:
But you are right I think we can remove it since we already have the `samplesheet_pe.csv` and `samplesheet_se.csv`, addressed here: [dd10518](https://github.com/nf-core/chipseq/pull/297/commits/dd105182f5cbca8fa61e2faa1c6f16d7b2855bf6)